.changeset/curly-weeks-talk.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
+---
+
+fixes emojis being parsed inside code blocks in text message attachment using `chat.postMessage` API call
```

Comment:
The issue also happened on the threads title, as the emojiParser is used there too, maybe we should add this info on the changeset


<img width="714" alt="Screenshot 2025-05-13 at 15 40 50" src="https://github.com/user-attachments/assets/908bb5de-669c-430e-a87f-05ae14f8770d" />

Reply:
This is not related to the issue itself... this is related to threads title not recognizing code blocks on it... must have a separated issue for it

Reply:
Unrelated, since sanitization for thread title is more deep.